bundles/azure-service-broker-0.0.1/README.md

```diff
@@ -0,0 +1,19 @@
+## Overview
+
+The Azure Service Broker bundle installs the [Open Service Broker for Azure](https://github.com/Azure/open-service-broker-azure) in a given Namespace.
+
+## Details
+
+The Azure Service Broker bundle contains one `default` plan.  
```

Comment:
point to github docs about azure service broker class (gcp broker has: https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/helm-broker-service-classes/gcp-broker-provider/docs/plans-details.md), where user can read more, for example will know that needs to provide a secret, l

Reply:
Just ask @piotrmiskiewicz how the URL will look like, add two links, to overview and details. PLease also fix GCP Bundle doc, to point to overiew and details.

Reply:
👍